### PR TITLE
 Use npc empath if no pc empaths are available

### PR DESCRIPTION
**Pull request by jandersson**
Fixes the bug in #2706 

I'll write some tests for safe-room to try and prevent this in the future.

### CODE REVIEW

File: safe-room.lic [hunk 1]
```diff
@@ -193,6 +182,22 @@ class SafeRoom
     bput('lie', 'You lie') if standing?
   end
 
+  def use_pc_empaths?(pc_empaths, settings)
+    return if pc_empaths.empty?
```
Comment (rpherbig) [inline safe-room.lic]: ❓  Should this be `return false`?

File: safe-room.lic [hunk 2]
```diff
@@ -32,18 +32,7 @@ class SafeRoom
...
-        .map { |room| pc_empaths.select { |empath| empath['id'] == room } }
-        .flatten
-        .each do |empath|
-        next unless DRRoom.pcs.include?(empath['name'].capitalize!) || in_game?(empath['name'])
-        next unless use_pc_empath?(empath['id'], empath['name'])
-        ensure_copper_on_hand(settings.safe_room_tip_threshold || 0, settings)
-        tip(settings.safe_room_tip_threshold, settings.safe_room_tip_amount, empath['name'])
-        break
-      end
+    if use_pc_empaths?(pc_empaths, settings)
```
Comment (rpherbig) [inline safe-room.lic]: 💡 What do you think about adding a comment to note this is deliberately empty?

File: safe-room.lic (hunk 1 above)
Comment (jandersson) [reply safe-room.lic]: Yea. im guessing predicates should be explicit about true/false

File: safe-room.lic (hunk 2 above)
Comment (jandersson) [reply safe-room.lic]: That sounds good to me.

**Comment by jandersson** [comment]
This should be good to go, I did some manual testing

**Pull request merged**